test/system/test_tpm2_createprimary.sh

```diff
@@ -52,7 +52,7 @@ cleanup
 
 # Keep the algorithm specifiers mixed to test friendly and raw
 # values.
-for gAlg in 0x04 sha256 0x0C; do
+for gAlg in sha1 0x04 sha256; do
```

Comment:
Same comment than before, sha1 is already tested (0x04). So just remove 0x0C (sha384).